src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx

```diff
@@ -0,0 +1,79 @@
+            <FeatureEnabledAccessOrNotFoundWrapper
+                policyID={policyID}
+                featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            >
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    testID={QuickbooksOutOfPocketExpenseConfigurationPage.displayName}
+                >
+                    <HeaderWithBackButton title={translate('workspace.qbo.exportExpenses')} />
+                    <ScrollView contentContainerStyle={styles.pb2}>
+                        {!isLocationEnabled && <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.qbo.exportOutOfPocketExpensesDescription')}</Text>}
+                        <OfflineWithFeedback pendingAction={pendingFields?.exportEntity}>
+                            <MenuItemWithTopDescription
+                                title={exportEntity ? translate(`workspace.qbo.${exportEntity}`) : undefined}
+                                description={translate('workspace.qbo.exportAs')}
+                                error={hasErrors && exportEntity ? translate(`workspace.qbo.${exportEntity}Error`) : undefined}
+                                onPress={() => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_OUT_OF_POCKET_EXPENSES_SELECT.getRoute(policyID))}
+                                brickRoadIndicator={hasErrors ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                                shouldShowRightIcon
+                            />
+                        </OfflineWithFeedback>
+                        {exportEntity === CONST.QUICKBOOKS_EXPORT_ENTITY.VENDOR_BILL && !isLocationEnabled && (
+                            <Text style={[styles.ph5, styles.mutedNormalTextLabel, styles.pt1, styles.pb2]}>{translate('workspace.qbo.exportVendorBillDescription')}</Text>
+                        )}
+                        {isLocationEnabled && <Text style={[styles.ph5, styles.mutedNormalTextLabel, styles.pt1]}>{translate('workspace.qbo.outOfPocketLocationEnabledDescription')}</Text>}
+                        {!isLocationEnabled && (
+                            <OfflineWithFeedback pendingAction={pendingFields?.exportAccount}>
+                                <MenuItemWithTopDescription
+                                    title={exportAccount}
+                                    description={translate('workspace.qbo.accountsPayable')}
+                                    onPress={() => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_OUT_OF_POCKET_EXPENSES_ACCOUNT_SELECT.getRoute(policyID))}
+                                    brickRoadIndicator={errors?.exportAccount ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                                    shouldShowRightIcon
+                                    error={errors?.exportAccount ? translate('common.genericErrorMessage') : undefined}
```

Comment:
Same question ^

Reply:
@s77rt i think for now it will be default one - nothing was specified in high level doc, and after fixing errorText we will move to api errors directly